lib/code-caller/code-caller-container.js

```diff
@@ -566,7 +562,8 @@ class CodeCallerContainer {
       try {
         await this.removeBindMount(this.hostDirectory.path);
       } catch (e) {
-        // Probably not mounted; swallow error
+        // Probably not mounted; swallow error.
+        logger.error('Error unmounting host directory', e);
```

Comment:
Would it be safer to only attempt `removeBindMount` here if `hasBindMount` is true? We could even move the check for `hasBindMount` into the start of `removeBindMout()` so that we can call it unconditionally.

The above code is clearly no worse than it currently is, so feel free to ignore this suggestion and merge.

Reply:
Yep, I made this change in ff9cac7.